lib/locales/pl.ts

```diff
@@ -233,6 +233,8 @@ export const locale: Locale = {
     divine_mercy_sunday: '2 Niedziela Wielkanocna czyli Niedziela Miłosierdzia Bożego',
     dominic_de_guzman_priest: 'Św. Dominika, prezbitera',
     easter_sunday: 'Niedziela Wielkanocy Zmartwychwstania Pańskiego',
+    // TODO: get a proper (non-google translate) translation
+    easter_vigil: 'Wigilia Wielkanocna',
```

Comment:
[src](https://drive.google.com/file/d/1KQ0XEoiQjX79sR1mfRc6OaRfeoulPcv5/view?usp=drive_link)

```suggestion
    easter_vigil: 'Wigilia Paschalna', // src: Kalendarz liturgiczny Diecezji Tarnowskiej, Wigilia Paschalna, p135 (https://drive.google.com/file/d/1KQ0XEoiQjX79sR1mfRc6OaRfeoulPcv5/view)
```

Reply:
i'll reformat w/ a block comment